x-pack/plugins/ml/public/jobs/jobs_list_new/components/jobs_list/job_description.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const COLORS = [
```

Comment:
Is there any way to use the EUI color variables directly, rather than using the HEX codes directly?

Reply:
not that i know of. the colors are in scss files